### ISSUE TITLE
[suggestion] Add Set section in Collections

### ISSUE THREAD

**Issue by dakeshi** [crowdsourcebooks/theswiftbook#29]
We have two section in the Collections: an array and a dictionary.
I think that the set type needs to be addressed in Collections.

What do you think about it?

**Comment by vandadnp** [comment]
I agree with you! Let's keep this issue open until someone adds that. Thoughts?
